CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 ## UNRELEASED
 
 ### Changed
+- ([#536](https://github.com/demos-europe/demosplan-ui/pull/536)) Move DpToggleForm to Core ([@ahmad-demos](https://github.com/@ahmad-demos))
```

Comment:
Two things regarding the changelog:

- This belongs into the "Removed" change group. (We follow the [common changelog](https://common-changelog.org/#24-change-group) convention).
- I suggest to write the entry from a more "generic" perspective. For users of the library it does not matter that we (as the library authors) decided to replicate the functionality somewhere else. The interesting fact is that we removed it here ("Remove  DpToggleForm component").

Reply:
Done in
https://github.com/demos-europe/demosplan-ui/pull/536/commits/97d250cc3af91365987171e5938b79398ceb2086
https://github.com/demos-europe/demosplan-ui/pull/536/commits/48edc93c457993533770a3adfab9d4feccfa1749